### DISCUSSION TITLE
Laravel 9.x not compatible with PHP 8.0.x

### DISCUSSION THREAD

**Discussion by nabeelgillani** [laravel/framework#44702] [Q&A]
- Laravel Version: 9.19.x
- PHP Version: 8.0.x
- Database Driver & Version:

### Description:

Laravel 9.x does not support PHP 8.0.x when you run composer install, it gives following error

Composer detected issues in your platform: Your Composer dependencies require a PHP version ">= 8.1.0". If you look at the composer.lock file I guess this is due to Symphony 6.1.

Where as Laaravel 9.x docs suggest it should work with PHP 8.0.x

**Comment by eduance** [comment]
This is not a bug related to Laravel, please open a support ticket at the discussion thread or seek help on Stackoverflow. Read the error message carefully and you'll find your way.

**Comment by lk77** [comment]
Laravel ask for symfony ^6.0, so your composer.lock was generated with php 8.1 at least

**Comment by nabeelgillani** [comment]
As per the docs it claims Laravel 9.x requires php 8.0 ... not the case it requires php 8.1... isnt that an issue?

**Reply by nabeelgillani** [reply]
specially causing problem for enterprise linux like RHEL 8

**Reply by lk77** [reply]
it does not require php 8.1, delete your composer.lock, it was generated with another php version
